salesagility/SuiteCRM
# hotfix-7.10.x -> hotfix merge (7.11.19)

## Description




- Cherry-picked and updated all 7.10.x commits to the hotfix branch.
- Includes 7.11.19 release.

## How To Test This

1. Check commit history.
2. Check Travis is passing.

## Types of changes


### Final checklist
